dkg-gadget/src/persistence.rs

```diff
@@ -278,7 +278,7 @@ where
 {
 	let rounds = worker.take_rounds();
 	let next_rounds = worker.take_next_rounds();
-	let time_to_restart = worker.get_time_to_restart(header);
+	let _time_to_restart = worker.get_time_to_restart(header);
```

Comment:
If we are not using the variable `_time_to_restart` can we just have `worker.get_time_to_restart(header);` with no return value? Thoughts?

Reply:
Alright